lms/djangoapps/discussion_api/api.py

```diff
@@ -328,12 +328,12 @@ def get_thread_list(
             })
 
     if following:
-        threads, result_page, num_pages = context["cc_requester"].subscribed_threads(query_params)
+        threads, result_page, num_pages, thread_count = context["cc_requester"].subscribed_threads(query_params)
```

Comment:
These four values keep getting passed around together.  Does it make sense to wrap them in a class or namedtuple to make them easier to keep track of, and easier to modify in the future?